src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -240,6 +240,8 @@ public function contains($key, $operator = null, $value = null)
     /**
      * Set realtion over each item.
      *
+     * @param  mixed  $name
```

Comment:
Why is this mixed?

Reply:
to support an array of relations, check second test case 
```php
$model->setRelation([
   'a' => $relationA,
   'b' => $relationB,
])
```

Reply:
so not mixed? `string|array`?

Reply:
make sense